cmd/juju/charmhub/download.go

```diff
@@ -178,46 +178,12 @@ func (c *downloadCommand) Run(cmdContext *cmd.Context) error {
 	if pSeries == "all" || pSeries == "" {
 		pSeries = version.DefaultSupportedLTS()
```

Comment:
If the user doesn't define a series/base on the cli, we should use the model config value in its place. With deploy having a model config default series/base is the same as using the flag on the cli for series/base. Fallback to the DefaultSupportedLTS.

Reply:
Unfortunately you can't, as this command is meant to be controller-less, so there won't be a model config for us to get.